WeakAuras/Types.lua

```diff
@@ -3229,6 +3229,12 @@ Private.noreset_swing_spells = {
   [35478] = true, -- Drums of Restoration
   [34120] = true, -- Steady Shot (rank 1)
   [19434] = true, -- Aimed Shot (rank 1)
+  [1464] = true, -- Slam (rank 1)
+  [8820] = true, -- Slam (rank 2)
+  [11604] = true, -- Slam (rank 3)
+  [11605] = true, -- Slam (rank 4)
+  [25241] = true, -- Slam (rank 5)
+  [25242] = true, -- Slam (rank 6)
```

Comment:
Did you check if the behavior is correct on classic_era?

Reply:
I have not. I sadly don't have a character available to test with there either without paying Blizzard for my classic era character.

So the safe options would be to either only include the new BCC version here (this is what the majority of people would care about), or generate it similarly to reset_swing_spells.